apps/api/src/app/shared/framework/response.decorator.ts

```diff
@@ -303,12 +303,11 @@ export const ApiResponse = <DataDto extends Type<unknown>>(
 
   return applyDecorators(
     ApiExtraModels(DataWrapperDto, dataDto),
-    responseDecoratorFunction(
-      options || {
-        description,
-        schema: buildSchema(shouldEnvelope, isResponseArray, dataDto),
-      }
-    )
+    responseDecoratorFunction({
+      description,
+      schema: buildSchema(shouldEnvelope, isResponseArray, dataDto),
+      ...options,
+    })
```

Comment:
This fixes a bug where swagger options was not working and overriding the schema definition